pom.xml

```diff
@@ -961,8 +933,8 @@
                                         <excludes>
                                             <exclude>module-info.java</exclude>
                                         </excludes>
-                                        <source>1.8</source>
-                                        <target>1.8</target>
+                                        <source>${java.version}</source>
+                                        <target>${java.version}</target>
```

Comment:
This probably is wrong. It does not need to be 1.8, it can be 11, but it should not be higher. When building Jersey, we use about three versions of JDK to produce proper multi-release jars.

Reply:
Actually this line is correct. `${java.version}` is used all over this file, and is *not* the JDK version, but actually is a *constant* defined in line 2157 of this file which explicitly locks to the particular value `11`, but not higher - just as you want it to be. :-)